src/web/aws-cloudwatch/KinesisStreams.jsx

```diff
@@ -2,20 +2,39 @@ import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import { Col, Row } from 'react-bootstrap';
 
-import { FormDataContext } from './context/FormData';
 import FormAdvancedOptions from './FormAdvancedOptions';
+import { FormDataContext } from './context/FormData';
+import { ApiContext } from './context/Api';
+import useFetch from './hooks/fetch';
 
 import FormWrap from '../common/FormWrap';
 import ValidatedInput from '../common/ValidatedInput';
-import Routes from '../common/Routes';
+import Routes, { ApiRoutes } from '../common/Routes';
+import { renderOptions } from '../common/Options';
 
 const KinesisStreams = ({ onChange, onSubmit }) => {
   const { formData } = useContext(FormDataContext);
+  const { availableStreams, setLogSample } = useContext(ApiContext);
+  const [fetchStatus, setFetchOptions] = useFetch(
```

Comment:
These need better names. What is actually fetched here? I guess it should be `[logSampleStatus, setLogSampleUrl]`?